locksmith/migrations/20230405185518-change-integer-to-big-integer.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query('CREATE DOMAIN pg_chain_id AS BIGINT;')
+    return Promise.all([
+      // Can't change as it is a primary key
+      // queryInterface.changeColumn('Blocks', 'chain', {
+      //   type: 'pg_chain_id',
+      // }),
+      queryInterface.changeColumn('Transactions', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('Locks', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('AuthorizedLocks', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('KeyMetadata', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('LockMetadata', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('UserTokenMetadata', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('StripeConnectLocks', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('Charges', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('LockIcons', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('LockMigrations', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('LockMigrations', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('ProcessedHookItems', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('PaymentIntents', 'chain', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('KeyRenewals', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('Verifiers', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('KeySubscriptions', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('ReceiptBases', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('Receipt', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('CustomEmailContents', 'network', {
+        type: 'pg_chain_id',
+      }),
+      queryInterface.changeColumn('LockSettings', 'network', {
+        type: 'pg_chain_id',
+      })
+    ])
+  },
+
+  async down(queryInterface, Sequelize) {
+    await Promise.all([
+      // Can't change as it is a primary key
+      // queryInterface.changeColumn('Blocks', 'chain', {
+      //   type: Sequelize.INTEGER,
+      // }),
+      queryInterface.changeColumn('Transactions', 'chain', {
+        type: Sequelize.INTEGER,
+      }),
+      queryInterface.changeColumn('Locks', 'chain', {
+        type: Sequelize.INTEGER,
```

Comment:
Why not use our new type for consistency so there is no mismatch in migration either?

Reply:
That is the `down` (revert) part of the migration so it goes back to its original state :)